store/kv/fdb.go

```diff
@@ -306,6 +313,10 @@ func (b *fbatch) SetVersionstampedValue(_ context.Context, _ []byte, _ []byte) e
 	return fmt.Errorf("batch doesn't support setting versionstamped value")
 }
 
+func (b *fbatch) SetVersionstampedKey(_ context.Context, _ []byte, _ []byte) error {
+	return fmt.Errorf("batch doesn't support setting versionstamped key")
```

Comment:
Should this return an error?

Reply:
realized after adding it in fdb layer that fdb is not returning an error for this. Makes sense, will remove it.

Reply:
Maybe we can just remove batch in one of the next diffs, it's unused anyway.